### PR TITLE
Fixes #86: TwicPics Components for React throws null ref error when u…

### LINKED ISSUE #86: TwicImg for React throws null ref error
Getting the following error using TwicImg in different layouts and screens in my app.

```
?v1&step=200:1 Uncaught TypeError: Cannot read properties of null (reading 'appendChild')
    at Be (?v1&step=200:1:3311)
    at ?v1&step=200:1:4719
    at Object.Y (?v1&step=200:1:2103)
    at ?v1&step=200:1:4878
    at ?v1&step=200:1:13591
    at ?v1&step=200:1:4883
    at Array.forEach (<anonymous>)
    at ?v1&step=200:1:153
    at ?v1&step=200:1:13553
    at NodeList.forEach (<anonymous>)
    at ?v1&step=200:1:153
    at ?v1&step=200:1:13758
    at Array.forEach (<anonymous>)
    at ?v1&step=200:1:153
    at S.X (?v1&step=200:1:13699)
```

Using `@twicpics/components` version 0.26.0, in a nextjs app dir project.
The error happens also in components that are purely client components.
It seems to reproduce on renders where an image was removed/added, doesn't happen on renders that don't change the images.

I found it hard to set up a working sandbox with the error just because of <TwicInstall /> and all, if there is some readymade sandbox you have I can try and reproduce in there.

### PR DESCRIPTION
**Pull request by mbgspcii**
…sing React StrictMode in development environment.

### CODE REVIEW

File: src/react/TwicInstall.tsx [hunk 1]
```diff
@@ -29,6 +29,8 @@ interface InstallAttributes {
     step?: number | string,
 }
 
+const SCRIPT_ELEMENT_ID = `__twicpics__DO_NOT_USE__`;
```
Comment (jaubourg) [inline src/react/TwicInstall.tsx:32]: Why not simply `twicpics-script` or something like this?

File: src/react/TwicInstall.tsx [hunk 2]
```diff
@@ -40,18 +42,19 @@ const TwicInstall: React.FC< InstallAttributes > = props => {
...
         "maxDPR": parseMaxDrp( props.maxDPR ),
         "path": parsePath( props.path ),
+        "scriptElementId": SCRIPT_ELEMENT_ID,
         "step": parseStep( props.step ),
     };
     setConfig( options );
     useEffect(
         () => {
-            register( options );
+            if ( !document.getElementById( SCRIPT_ELEMENT_ID ) ) {
```
Comment (jaubourg) [inline src/react/TwicInstall.tsx:51]: Not `options.scriptElementId`. I'm confused as to why we have an option if it's not taken into account.

File: src/react/TwicInstall.tsx (hunk 1 above)
Comment (mbgspcii) [reply src/react/TwicInstall.tsx:32]: Ok i'll make the update

File: src/react/TwicInstall.tsx (hunk 2 above)
Comment (mbgspcii) [reply src/react/TwicInstall.tsx:51]: This option is taken into account in the register method.
But you are right. We can use options.scriptElementId here too.

**Pull request merged**